internal/pkg/upgrade/upgrade.go

```diff
@@ -62,21 +64,22 @@ func Upgrade(continueDirectly bool) error {
 		return err
 	}
 
-	ltstReleaseTagName, err := ghClient.GetLatestReleaseTagName()
+	//ltstReleaseTagName, err := ghClient.GetLatestReleaseTagName()
+	lastReleaseTagName, err := ghClient.GetLatestReleaseTagName()
```

Comment:
`ltst` is short for `latest` so there are no misspellings here.

But there is really something that needs to be changed here, and we should get the latest version directly through s3, not github.

You can `GET` this URL to get the latest version from s3: `https://download.devstream.io/latest_version`

Reply:
I will pay attention to it, I will look at it later😀

Reply:
> > `ltst` is short for `latest` so there are no misspellings here.
> > But there is really something that needs to be changed here, and we should get the latest version directly through s3, not github.
> > You can `GET` this URL to get the latest version from s3: `https://download.devstream.io/latest_version`
> 
> @aFlyBird0 I think the `ltst` is a bad abbreviation. How about leave `latest` here?

lgtm